config.go

```diff
@@ -39,28 +39,8 @@ type Config struct {
 	Providers map[string]ProviderConstructor `json:"-"`
 }
 
-// HeaderConfig specifies relevant header parameters.
-type HeaderConfig struct {
-
-	// SpanIDHeaderName is used to set custom header in response and logs.
-	SpanIDHeaderName string `json:"spanIDHeaderName"`
-
-	// TraceIDHeaderName is used to set custom header in response and logs.
-	TraceIDHeaderName string `json:"traceIDHeaderName"`
-}
-
 // TraceConfig will be used in TraceMiddleware to use config and TraceProvider
 // objects created by ConfigureTracerProvider.
 type TraceConfig struct {
-	HeaderConfig  HeaderConfig
 	TraceProvider trace.TracerProvider
 }
-
-// TracingConfig represents a component for tracing. It will be used to unmarshal
-// the configuration provided in application.yaml file.
-type TracingConfig struct {
```

Comment:
Do we foresee adding a different section similar to provider? If so, it might be worth keeping `TracingConfig` so we can standardize on how we read tracing options from our config files.

Reply:
if we want to read multiple configs from the same key in the config then it will use, as of now i don't see we are reading multiple configs under the tracing

This will be the structure  at the client-side 
```
# tracing provides configuration for OpenTelemetry
tracing:
  # provider is the provider name. Currently, stdout, jaegar and zipkin are supported.
  provider: "stdout"

  # skipTraceExport only applies when provider is stdout. Set skipTraceExport to true
  # so that trace information is not written to stdout.
  skipTraceExport: true

  # endpoint is where trace information should be routed. Applies to zipkin and jaegar.
  # endpoint: "http://localhost:9411/api/v2/spans"
```